homeassistant/bootstrap.py

```diff
@@ -83,6 +83,14 @@ def async_from_config_dict(config: Dict[str, Any],
     This method is a coroutine.
     """
     start = time()
+
+    if enable_log:
+        async_enable_logging(hass, verbose, log_rotate_days, log_file)
+
+    if sys.version_info[:2] < (3, 5):
+        _LOGGER.warning('Python 3.4 support has been deprecated and will be '
```

Comment:
Maybe rephrase this to represent the change for the common user a bit more?
`Python 3.4 support has been deprecated and will be`
`removed in the beginning of 2018.`
` Please upgrade Python or your chosen distribution.`

Reply:
Also, not sure if this is overkill, but it would be nice to link to a blog post or a doc where we can explain why support is being dropped, and direct users to further resources for different distros. Especially as we get a broader audience, not everyone is going to understand what this means.

Reply:
The why of dropping of support is not super important to me as opposed to how I can to switch to Python 3.5, etc.

Reply:
For the platforms there are already install guides, it would be good to detail whether an upgrade is possible, and if so how.

Reply:
We can't add upgrade instructions for every platform. I'll add a link to the blog post and the blog post will contain instructions for Hassbian. If people don't use our recommended platforms, they are on their own.